content/blog/2022/11/2022-11-14-hacktoberfest-recap.adoc

```diff
@@ -0,0 +1,159 @@
+Jagruti is Based in Mumbai, India and active in various areas of open source.
+This immersion is the main component of their strategy to explore and learn new things.
+They are not afraid of challenges and have received assistance from the friendly, open-source communities. 
+They are inspired by thought leaders such as Ankita Tripathi.
+Jagruti's contributions to Jenkins during Hacktoberfest includes documentation, code examples, and testing.
+They have also provided reviews for other community members, which supports the open-source spirit.
+
+We also want to welcome link:https://github.com/highflyer910[Thea Mushambadze].
+Thea created three new logos: link:https://www.jenkins.io/artwork/#turkey[Turkey], link:https://www.jenkins.io/artwork/#georgia[Georgia], and a link:https://www.jenkins.io/artwork/#nerd[Jenkins nerd] themed logo.
+These logos help connect Jenkins to users globally and extend the community even further.
+Thea is a freelance developer, and this is their first time contributing to Jenkins in Hacktoberfest.
+Their motivation when starting was curiosity and learning more about open source.
+Through their contributions, they have learned how to work with someone else's code.
+There have been challenges when new tools are installed, but don't work as expected.
+However, these are learning experiences, as each one is an opportunity to understand what is causing the behavior.
+
+image:/images/post-images/2022-11-14-hacktoberfest-recap/thea.png[Thea Mushambadze, width=20%]
+
+Next, we want to introduce link:https://github.com/OmkarBorhade98[Omkar Borhade] from Pune, India.
+Omkar just started as a software engineer, after graduating as a Mechanical Engineer.
+They are new to open source, and this was their first time participating in Hacktoberfest.
+Their goal was to become more familiar the Jenkins project.
+This was acheived by a lot of reading, approaching issues with an open mind, and providing documentation.
+As a new contributor, they shared it can be difficult to understand the issue and what is needed for resolution.
+This knowledge comes with experience, so have some patience when getting started.
+
+image:/images/post-images/2022-11-14-hacktoberfest-recap/omkar.png[Omkar Borhade, width=20%]
+
+link:https://github.com/krisstern[Kris Stern] is a full-stack software engineer based out of Hong Kong SAR, China.
+They have previously trained as an observational astrophysicist with a PhD in physics, and is currently studying part-time for a computer science master's degree. 
+This is their third Hacktoberfest, and first Hacktoberfest contributing to Jenkins.
+They have contributed to open source previously for astronomy, but shifted focus since graduating.
+Their goals during Hacktoberfest were giving back to the Jenkins community and acquiring more software development skills along the way.
+
+
+
+== Returning Jenkins contributors
+
+First, we want to welcome back link:https://github.com/jimklimov[Jim Klimov].
```

Comment:
Since this is relatively deep in the document, I hesitate to use the word "First"
```suggestion
link:https://github.com/jimklimov[Jim Klimov] is a regular Jenkins contributor.
```